psst-gui/src/ui/mod.rs

```diff
@@ -129,13 +128,14 @@ fn root_widget() -> impl Widget<AppState> {
         .with_default_spacer()
         .with_child(user::user_widget())
         .center()
-        .fix_height(100.0)
+        .fix_height(88.0)
         .background(Border::Top.with_color(theme::GREY_500));
 
     let sidebar = Flex::column()
         .with_flex_child(playlists, 1.0)
         .with_child(controls)
-        .background(theme::BACKGROUND_DARK);
+        .background(Border::Right.with_color(theme::GREY_500))
+        .fix_width(185.0);
```

Comment:
I think it would still be nice to have this dynamically resize. Is there something preventing this?

Reply:
I ran into difficulties with having it dynamically resize due to the split function still giving space to have a widget on the right even when the queue widget has disappeared.

Reply:
I just realised that I just need to not use the split widget and I can just use the flex child! I will change now!

Reply:
Right! Good sleuthing!